packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/PaymentMethods/Payments/PaymentCard/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { fiatToString } from 'core/exchange/currency'
+import { FiatType, SBPaymentMethodType } from 'core/types'
+import { FormattedMessage } from 'react-intl'
+import { Icon } from 'blockchain-info-components'
+import { Title } from 'components/Flyout'
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+
+const DisplayContainer = styled.div`
+  display: flex;
+  width: 100%;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 16px 40px;
+  flex-direction: row;
+  cursor: pointer;
+  border-bottom: 1px solid ${props => props.theme.grey000};
+  &hover {
+    background-color: ${props => props.theme.grey100};
+  }
+`
```

Comment:
This component is reused in 4 places, please create and move to a nearby `components` folder